provider/cmd/pulumi-resource-random/schema.json

```diff
@@ -227,151 +183,162 @@
                     "keepers": {
                         "type": "object",
                         "additionalProperties": {
-                            "$ref": "pulumi.json#/Any"
+                            "type": "string"
                         },
-                        "description": "Arbitrary map of values that, when changed, will\ntrigger a new id to be generated. See\nthe main provider documentation for more information.\n",
-                        "willReplaceOnChanges": true
+                        "description": "Arbitrary map of values that, when changed, will trigger recreation of resource. See the main provider documentation for more information.\n"
                     },
                     "max": {
-                        "type": "integer",
-                        "description": "The maximum inclusive value of the range.\n",
-                        "willReplaceOnChanges": true
```

Comment:
willReplaceOnChanges loss is documentation-only, I will make a ticket but it's not blocking.